packages/office-addin-dev-settings/README.md

```diff
@@ -252,10 +252,10 @@ Switches the webview runtime in Office for testing and development scenarios.
 
 Syntax:
 
-`office addin-dev-settings webview <manifest> <runtime>`
+`office addin-dev-settings webview <manifest> <web-view-type>`
 
 `manifest`: path to manifest file. 
 
-`runtime`: Office runtime to load (currently accepts 'ie', 'edge', 'edgechromium', or 'default'). 
+`runtime`: Office webview to load (currently accepts 'ie', 'edge' or 'edge-chromium, 'edge-legacy', or 'default'). 
```

Comment:
I noticed a slight tweak that could be make here, can you remove `currently accepts` so inside parentheses is just:
```
(`edge` or 'edge-chromium', 'edge-legacy`, `ie`, or `default`)
```

Once we decide whether `edge` is for `edge-chromium` or `edge-legacy`, you can put it with the appropriate string.

I think it helps to reorder them from most recent to oldest.